Rick-and-Morty/Rick And Morty/Model/Character.swift

```diff
@@ -8,6 +8,19 @@
 
 import Foundation
 
+
+struct CharactersList: Decodable {
```

Comment:
Id probably update the naming here as it isnt a list. Maybe just `CharactersResponse` is my usual go to for this kind of base level json response.